MANIFEST.in

```diff
@@ -3,3 +3,6 @@ graft wagtail
 prune wagtail/wagtailadmin/static_src
 global-exclude __pycache__
 global-exclude *.py[co]
+global-exclude *.stories.js
+global-exclude *.stories.tsx
+recursive-exclude wagtail/ *.md
```

Comment:
will we also need to exclude client/ *md ? or will we not used md files for those stories?
Note: there is a md file we may want to keep though `client/README.md`

Maybe client does not get included at all so non-issue (I am still learning the Manifest syntax)

Reply:
Yep, `client` isn’t included at all currently so we should be good.